packages/web-app-draw-io/src/App.vue

```diff
@@ -125,6 +125,7 @@ export default defineComponent({
       }
     },
     load() {
+      this.checkPermissions()
```

Comment:
load isnt async and await is missing

Reply:
Not needed. `checkPermissions` and `getFileContents` can run in parallel. No need to wait until `checkPermissions` is finished.

Reply:
Pushed a commit to make that easier to see